app/konnect/analytics/reference.md

```diff
@@ -0,0 +1,107 @@
+-------|------------
+Request Count | Total number of API calls within the selected time frame. This includes requests that
+were rejected due to rate limiting, failed authentication, and so on.
+Requests per Minute | Number of API calls per minute within the selected time frame.
+Response Latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when {{site.base_gateway}} receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
+Request Size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
+Response Size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
+
+{:.note}
+> **Note:** {{site.konnect_saas}} Analytics uses percentiles to enable you to understand the real performance characteristics of your APIs. Percentiles depict a more accurate picture of what most end users experience using your API instead of hiding critical experiences by displaying averages.
+
+## Grouping and filtering
+
+When selecting a category to filter or group by, you have the following options:
+
+Category | Description
+---------|------------
+None | Aggregate all of the data in the organization without any grouping.
+Service | Group or filter the data by {{site.konnect_short_name}} service.
+Service Version | Group or filter the data by {{site.konnect_short_name}} service version. <br><br>If you select multiple service versions in a report, the report shows the sum of requests for all selected versions broken down by service. It **does not** show data points for individual service versions.
+Route | Group or filter the data by route. Route names are composed of elements referencing their related service, service version, and runtime group. See [Route entity format](#route-entity-format) for details.
+Application | Group or filter the data by application.
+Status Code | Group or filter the data by individual response status code. Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx,
+  4xx, 5xx). The Status Codes view contains the counts of status code classes
+  graphed over time, as well as the ratio of code classes to total requests.
+Status Code (grouped) | Group or filter the data by response status code category: 1XX, 2XX, 3XX, 4XX, and 5XX.
```

Comment:
```suggestion
Status Code | Group or filter the data by individual response status code.
Status Code (grouped) | Group or filter the data by response status code category: 1XX, 2XX, 3XX, 4XX, and 5XX. Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx,
  4xx, 5xx). The Status Codes view contains the counts of status code classes
  graphed over time, as well as the ratio of code classes to total requests.
```
@adorack That's my bad, thanks for catching that! Is this correct now?

Reply:
Ah, no worries.  That's better... I don't think we need the last sentence ("The Status Codes view contains the counts of status code classes graphed over time, as well as the ratio of code classes to total requests.")?  I can't think of anything offhand that explicitly graphs the ratio of code classes to total requests -- the summary page shows the ratio of error classes to total requests, but not individual code classes.  Reports definitely don't have this functionality.

To my eyes, the second sentence (Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx, 4xx, 5xx).) is redundant, but that's more of a nitpick.

Reply:
This section
> 1XX, 2XX, 3XX, 4XX, and 5XX. Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx,
is mixed case. 

I also think the second sentence can be removed.